lib/ansible/modules/cloud/ovh/ovh_monthly_billing.py

```diff
@@ -34,22 +34,40 @@
         type: str
         description:
             - ID of the instance, get it with U(https://api.ovh.com/console/#/cloud/project/%7BserviceName%7D/instance#GET)
+    endpoint:
+        type: str
+        description:
+            - The endpoint to use (for instance ovh-eu)
+    application_key:
+        type: str
+        description:
+            - The applicationKey to use
+    application_secret:
+        type: str
+        description:
+            - The application secret to use
+    consumer_key:
+        type: str
+        description:
+            - The consumer key to use
 '''
 
 EXAMPLES = '''
-# basic
+# basic usage, using auth from /etc/ovh.conf
   - ovh_monthly_billing:
        project_id: 0c727a20aa144485b70c44dee9123b46
        instance_id: 8fa89ad2-8f08-4220-9fa4-9695ea23e948
 
-# more complex
+# a bit more more complex
+  # get openstack cloud ID and instance ID, OVH use them in it's API
   - os_server_facts:
```

Comment:
os_server_facts is deprecated : https://docs.ansible.com/ansible/latest/modules/os_server_facts_module.html

Reply:
fixed